src/lib/utils/renderTemplate.ts

```diff
@@ -1,17 +1,27 @@
-import hogan from 'hogan.js';
+import hogan, { HoganOptions, Template } from 'hogan.js';
+import { Templates, HoganHelpers } from '../../types';
+import { BindEventForHits } from './createSendEventForHits';
+
+type ActualHoganHelpers = {
```

Comment:
That seems to better represent `transformHelpersToHogan`:

```suggestion
type TransformedHoganHelpers = {
```